docs/stylesheets/extra.css

```diff
@@ -50,6 +56,33 @@
   --md-admonition-bg-color: var(--md-default-bg-color);
 }
 
+/* Header color */
+.md-header {
+    background-color: var(--md-default-bg-color);
+    color: var(--md-typeset-color);
+}
+.md-tabs {
+    background-color: var(--md-default-bg-color);
+    color: var(--md-typeset-color);
+    border-bottom: 1px solid var(--md-border-color);
+}
+.md-search__form:hover {
+    background-color: #0000001f;
+}
```

Comment:
Usually tabs = 2 spaces in CSS, the formatting is uneven in this file (you may want to turn on auto-format on save in your IDE).